public/app/features/explore/state/actions.ts

```diff
@@ -451,23 +431,24 @@ export const runQueries = (exploreId: ExploreId): ThunkResult<void> => {
 
     // Some datasource's query builders allow per-query interval limits,
     // but we're using the datasource interval limit for now
-    const minInterval = datasourceInstance.interval;
+    const minInterval = datasourceInstance?.interval;
```

Comment:
Why would there ever not be an instance?

Reply:
Good question, but it's defined as an optional property so the `?` is needed to suppress a strict null error.